2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_setup.py:_flush():76] Configure stats pid to 2416
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_setup.py:_flush():76] Loading settings from C:\Users\MNL431\.config\wandb\settings
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_setup.py:_flush():76] Loading settings from C:\Users\MNL431\Desktop\traffic_DQN\wandb\settings
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'training_main.py', 'program_abspath': 'C:\\Users\\MNL431\\Desktop\\traffic_DQN\\training_main.py', 'program': 'C:\\Users\\MNL431\\Desktop\\traffic_DQN\\training_main.py'}
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\MNL431\Desktop\traffic_DQN\wandb\run-20240411_030127-gyyhqj0d\logs\debug.log
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\MNL431\Desktop\traffic_DQN\wandb\run-20240411_030127-gyyhqj0d\logs\debug-internal.log
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_init.py:init():566] calling init triggers
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'architecture': 'DQN', 'dataset': 'CIFAR-100', 'epochs': 10000}
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_init.py:init():616] starting backend
2024-04-11 03:01:27,670 INFO    MainThread:2416 [wandb_init.py:init():620] setting up manager
2024-04-11 03:01:27,695 INFO    MainThread:2416 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-04-11 03:01:27,701 INFO    MainThread:2416 [wandb_init.py:init():628] backend started and connected
2024-04-11 03:01:27,723 INFO    MainThread:2416 [wandb_init.py:init():720] updated telemetry
2024-04-11 03:01:27,759 INFO    MainThread:2416 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-04-11 03:01:28,321 INFO    MainThread:2416 [wandb_run.py:_on_init():2262] communicating current version
2024-04-11 03:01:28,365 INFO    MainThread:2416 [wandb_run.py:_on_init():2271] got version response upgrade_message: "wandb version 0.16.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-04-11 03:01:28,365 INFO    MainThread:2416 [wandb_init.py:init():804] starting run threads in backend
2024-04-11 03:01:31,385 WARNING MainThread:2416 [wandb_init.py:init():1198] interrupted
Traceback (most recent call last):
  File "C:\Users\MNL431\Anaconda3\envs\cartpoletest\lib\site-packages\wandb\sdk\wandb_init.py", line 1176, in init
    run = wi.init()
  File "C:\Users\MNL431\Anaconda3\envs\cartpoletest\lib\site-packages\wandb\sdk\wandb_init.py", line 817, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "C:\Users\MNL431\Anaconda3\envs\cartpoletest\lib\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "C:\Users\MNL431\Anaconda3\envs\cartpoletest\lib\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "C:\Users\MNL431\Anaconda3\envs\cartpoletest\lib\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "C:\Users\MNL431\Anaconda3\envs\cartpoletest\lib\threading.py", line 581, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\MNL431\Anaconda3\envs\cartpoletest\lib\threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-04-11 03:01:36,069 WARNING MsgRouterThr:2416 [router.py:message_loop():77] message_loop has been closed
